src/utils/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const ncNewsApi = axios.create({
+  baseURL: "https://nc-news-v54w.onrender.com/api",
+});
+
+export const getAllArticles = () => {
+  return ncNewsApi
+    .get("/articles")
+    .then((res) => {
+      return res.data.articles;
+    })
+    .catch((err) => {
```

Comment:
You dont need a catch in here, or you'll have two catches. Only need the catches in the component where its being used